.gitignore

```diff
@@ -6,3 +6,4 @@ results*
 *.log
 tools/android-sdk-linux
 ipynb/*.pid
+/vagrant
```

Comment:
I think the `/` should be there - if there's a random file called `vagrant` anywhere else in the tree we shouldn't be ignoring it. 

However I've just noticed it's inconsistent with the PID one I just added and the android-sd-linux one. How about I stick a `/` at the beginning of all of them?